cmd/gitserver/server/server.go

```diff
@@ -2261,17 +2261,15 @@ func (s *Server) doClone(ctx context.Context, repo api.RepoName, dir common.GitD
 	pr, pw := io.Pipe()
 	defer pw.Close()
 
-	redactor := newURLRedactor(remoteURL)
-
 	go readCloneProgress(s.DB, logger, newURLRedactor(remoteURL), lock, pr, repo)
 
 	output, err := runRemoteGitCommand(ctx, s.RecordingCommandFactory.Wrap(ctx, s.Logger, cmd), true, pw)
-
 	// best-effort update the output of the clone
-	go s.setLastOutput(context.Background(), repo, redactor.redact(string(output)))
+	redactedOutput := newURLRedactor(remoteURL).redact(string(output))
+	go s.setLastOutput(context.Background(), repo, redactedOutput)
```

Comment:
```suggestion
    // best-effort update the output of the clone
	go s.setLastOutput(context.Background(), repo, redactedOutput)
```

We need to move the comment to where it's appropriate.